lib/ipam/ipam.go

```diff
@@ -218,7 +158,7 @@ func (c ipams) autoAssign(num int, handleID *string, attrs map[string]string, po
 		}
 
 		if retries == 0 {
-			return nil, goerrors.New("Max retries hit")
+			return nil, errors.New("Max retries hit")
```

Comment:
can we add some more context into this error? Max retries of what

Reply:
we can - but it's old code, it's an unusual error condition, it's in multiple places and I'm not sure it gains us anything useful.